pipeline/frontend/yaml/parse.go

```diff
@@ -37,3 +45,19 @@ func ParseString(s string) (*types.Workflow, error) {
 		[]byte(s),
 	)
 }
+
+func checkVersion(b []byte) (int, error) {
+	ver := struct {
+		Version int `yaml:"version"`
+	}{}
+	_ = xyaml.Unmarshal(b, &ver)
+	if ver.Version == 0 {
+		// default: version 1
+		return 1, nil
```

Comment:
can we make this var const in "shared/const" ?

Reply:
Probably version should be type/const/enum